### DISCUSSION TITLE
Trying to determine how to represent a definition in the "mind map" spirit

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1486] [Help]
<img width="912" alt="freeeplane option 1 for how to show -equals to-" src="https://github.com/freeplane/freeplane/assets/147225942/caf59d38-b6e5-443f-a256-ed3310227bb3">
<img width="722" alt="freeeplane option 2 for how to show -equals to-" src="https://github.com/freeplane/freeplane/assets/147225942/b0a9e0cb-75a1-4ca3-9395-8a89328e0c61">

The full definition is: The Unclean Hands doctrine bars a plaintiff from bringing a lawsuit if the plaintiff has acted unethically in respect to the subject of the claim. 
As you can see, I have broken down that definition into several nodes. However, I am still trying to figure out how to best represent that definition within the "mind map" formula.
In option 1, I kept everything linear. If I keep this option, I will need to figure out if there is a way to put an "equals" sign before "judgment barred."
In option 2, I made the definition flow vertically, and then used a summary node to emphasize the final result.
In all, I'm just throwing out ideas for how to do this. If anyone has any suggestions or thoughts, please share

**Comment by SchneiderCoppola** [comment]
Greetings, James here.

This is my area of expertise, and I currently current a youtube freeplane tutorial channel.

Maybe we could kill two birds with on stone.

I'm interested in trying to show you way ways to deal with that.

Would be good If we could video call to deal with this matter.

I'm also from law school, but civil law, and I didn't studied law in english.

But I did a really good law school.

But my expertise is really good in mind maps

Here is a few things that would be very helpful for you:
 - Universal Hierarchy of Knowledge
 - Key Words 
 -  Optimize vertical and horizontal display of the information 
 - Smart Drawering 
  
  I would need a video call of at least 1 hour so we could deal with it
  
  But I promise it will worth it, Will be very productive
  
  And I believe the whole Freeplane Community can benefit from it too
  
  I'm very proudly watching the Downloads reach 20k / 30k a Week
  
  I'm gonna drop my e-mail here so we can try to arrange something
  schneidercoppola@gmail.com
  
  Reach me out

**Reply by adamfreeplaneuser01** [reply]
Very interesting! I'd be down to make this happen.
What is the name of your Youtube channel?

**Reply by SchneiderCoppola** [reply]
https://www.youtube.com/FreeplaneEnglish

E-mail me at
schneidercoppola@gmail.com

Let's set up a day and time

**Reply by adamfreeplaneuser01** [reply]
Gotcha! I am currently in study mode for finals coming up this week, so as soon as that's all done I'll email you and we'll make it happen

**Reply by dpolivaev** [reply]
@adamfreeplaneuser01 @SchneiderCoppola Hello folks, I am curious if you could meet and if anything came out.

**Reply by adamfreeplaneuser01** [reply]
@dpolivaev @SchneiderCoppola I have one more school final at the end of this month (November) After that, we will work out a time to make it happen!
